src/state/json-diff-worker.js

```diff
@@ -0,0 +1,67 @@
+import { DiffPatcher } from "jsondiffpatch";
+import { v4 as uuidv4 } from "uuid";
+
+export class JsonDiffWorker {
+  queue = new Map();
+
+  constructor() {
+    try {
+      const workerAdress = require("worker-plugin/loader!./json-diff.worker.js");
```

Comment:
Changed this so that consumers create the web worker instance, e.g.

```js
import applyDevTools from 'prosemirror-dev-tools';

applyDevTools(editorView, {
  diffWorker: new Worker("prosemirror-dev-tools/json-diff.worker", {
    type: "module",
  }),
});
```

Reply:
This is still strictly optional and will fall back to previous behaviour